src/collector/ActionBatchingCollector.js

```diff
@@ -0,0 +1,30 @@
+
+const localStorageActionsKey = 'eggplantRciActions';
+export default class ActionBatchingCollector {
+
+  constructor(localStorage) {
+    this.localStorage = localStorage;
+  }
+
+  /**
+       * @param  {Event} event
+       * @param  {Context} context
+       */
+  async prepare (event, context) {
+    /** Pull out the actions from local storage */
+    const actions = this.localStorage.getItem(localStorageActionsKey);
+
+    /** Append the new action */
+    actions.push(event.eventInfo5[0]);
+    if (context.scope === 'state') {
```

Comment:
@robertdumitrescu MAINTAINABILITY - I'd rather we use ENUMs than lose strings.  We have some set up here: https://github.com/TestPlant/real-user-data-sdk-js/blob/development/src/core/constants.js#L1